src/dashboard/src/components/ingest/views.py

```diff
@@ -500,6 +573,86 @@ def transfer_backlog(request):
     return helpers.json_response(return_list)
 
 
+@decorators.elasticsearch_required()
+def transfer_appraisal(request):
```

Comment:
Instead of almost duplicating transfer_backlog, maybe make it a parameter that selects between the different functions?  Could be done in the urls.py.  A bit messier now, but then we don't have to worry about applying fixes to both, and we're planning on removing transfer_backlog eventually anyway.